solvers/vu_condat_cd.py

```diff
@@ -0,0 +1,56 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from cd_solver.sklearn_api import Lasso
+
+
+class Solver(BaseSolver):
+    name = 'vu-condat-cd'
+
+    install_cmd = 'conda'
+    requirements = [
+        'pip:git+https://github.com/Badr-MOUFAD/Fercoq-Bianchi-solver.git@sklearn-api'
+    ]
+
+    references = [
+        "Olivier Fercoq and Pascal Bianchi, "
```

Comment:
You can add this paper too https://hal.archives-ouvertes.fr/hal-01941152v2/document that is more dedicated to the package itself